dash/testing/browser.py

```diff
@@ -157,15 +161,17 @@ def _get_firefox():
 
     def get_logs(self):
         """get_logs works only with chrome webdriver"""
-        return (
-            [
-                entry
-                for entry in self.driver.get_log("browser")
-                if entry["timestamp"] > self._last_ts
-            ]
-            if self.driver.name == "chrome"
-            else []
-        )
+        if self.driver.name == 'Chrome':
+            return (
+                [
+                    entry
+                    for entry in self.driver.get_log("browser")
+                    if entry["timestamp"] > self._last_ts
+                ]
+            )
+        else:
+            warnings.warn("get_logs always return None with your webdriver")
+            return None
```

Comment:
Ah, that's a nice compromise - so you can do eg:
`assert not self.get_logs()`
and you'll get the warning but not a test failure. But if you ask for something more explicit like
`assert self.get_logs() == []`
the test will actually fail (along with the warning that will show you *why* it failed)

Small point: perhaps explicitly say "with ~your webdriver~ webdrivers other than Chrome"